### PR TITLE
fix: overwrite kernel args from --extra-agent-kernel-args

### LINKED ISSUE #251: serial console setting does not work with --extra-agent-kernel-args
By default, agent environment kernel parameters has `console=ttyS0`.
With newly added `--extra-agent-kernel-args`, enabled the ability to add  `console=ttyS1`. (the config working in my servers)
The implemenation ended up with both parameters present, I'm not sure how kernel handles this, but in real test, I still could not get serial console output during agent running.

From from https://www.kernel.org/doc/html/v5.9/admin-guide/serial-console.html, below is mentioned:

    Note that you can only define one console per device type (serial, video).

### PR DESCRIPTION
**Pull request by smira**
This allows to override default options (e.g. `console=ttyS0`) with
provided values. Make sure that full value is given for each option, as
any default values for the option are dropped if specified in
`--extra-agent-kernel-args`, e.g.
`--extra-agent-kernel-args "console=tty0 console=ttyS1 panic=5"`.

See also talos-systems/go-procfs#5

Fixes #251

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

### CODE REVIEW

**Review by Unix4ever — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -9,7 +9,7 @@ MODULE := $(shell head -1 go.mod | cut -d' ' -f2)
 
 ARTIFACTS := _out
 PKGS ?= ./...
-TALOS_RELEASE ?= v0.7.0
```
Comment (smira) [inline Makefile:12]: this is side-effect of pulling in new go-procfs with `.Get()` fix - it also pulls in new default KSPP args which are not compatible with 0.7

I have an issue in our backlog to decouple default args from go-procfs

File: app/metal-controller-manager/internal/ipxe/ipxe_server.go [hunk 2]
```diff
@@ -50,7 +52,7 @@ exit
 
 var (
 	apiEndpoint          string
-	extraAgentKernelArgs []string
```
Comment (smira) [inline app/metal-controller-manager/internal/ipxe/ipxe_server.go:53]: change format to whatever kernel args format is (with spaces), as commas have special meaning in kernel args

File: hack/scripts/integration-test.sh [hunk 3]
```diff
@@ -18,9 +18,6 @@ function build_registry_mirrors {
 
       REGISTRY_MIRROR_FLAGS="${REGISTRY_MIRROR_FLAGS} --registry-mirrors ${registry}=http://${addr}:5000"
     done
-
-    local addr=`python3 -c "import socket; print(socket.gethostbyname('registry.ci.svc'))"`
```
Comment (smira) [inline hack/scripts/integration-test.sh:22]: this is long gone, just gc'ed it

File: hack/scripts/integration-test.sh [hunk 4]
```diff
@@ -38,5 +35,5 @@ ${PREFIX} "${INTEGRATION_TEST}" test integration \
     --talosctl-path "${TALOSCTL}" \
     --clusterctl-config "${CLUSTERCTL_CONFIG}" \
     --power-simulated-explicit-failure-prob=0.1 \
-    --power-simulated-silent-failure-prob=0.1 \
```
Comment (smira) [inline hack/scripts/integration-test.sh:41]: brought it to zero as it introduces flakes and we don't need them

**Review by Ulexus — APPROVED** [review]

**Pull request merged**